cache/src/main/java/io/envoyproxy/controlplane/cache/ConfigWatcher.java

```diff
@@ -20,10 +20,12 @@
    * @param request the discovery request (node, names, etc.) to use to generate the watch
    * @param knownResourceNames resources that are already known to the caller
    * @param responseConsumer the response handler, used to process outgoing response messages
+   * @param hasClusterChanged indicates if EDS should be send. Works in ads mode only.
```

Comment:
`should be sent`

Reply:
I would also change it to 
> Indicates if EDS should be sent immediately, even if version has not been changed. Works in ADS mode only.

even better if @deejay1 could check if spelling/grammar in this sentence is correct :)